UnitystationLauncher/UnitystationLauncher.csproj

```diff
@@ -101,25 +101,35 @@
     <AvaloniaResource Remove="Views\ForgotPasswordView.xaml" />
     <AvaloniaResource Remove="Views\HubUpdateView.xaml" />
     <AvaloniaResource Remove="Views\InstallationsPanelView.xaml" />
+    <AvaloniaResource Remove="Views\NewsPanelView.xaml" />
     <AvaloniaResource Remove="Views\ServersPanelView.xaml" />
+    <AvaloniaResource Remove="Views\SettingsPanelView.xaml" />
   </ItemGroup>
   <ItemGroup>
+    <None Remove="Assets\BG_News.png" />
+    <None Remove="Assets\button_context.png" />
+    <None Remove="Assets\leftbutton.png" />
+    <None Remove="Assets\line.png" />
+    <None Remove="Assets\rightbutton.png" />
+    <None Remove="Assets\unity_ico.png" />
```

Comment:
Can you make the naming more consistent? some of these use snake case: `one_two`, some don't use any scheme: `onetwo`, and some use some kind of pascal case: `One_Two`

Reply:
Pascal_Snake